src/relay/ir/pretty_printer.cc

```diff
@@ -343,13 +347,18 @@ class PrettyPrinter :
   }
 
   bool IsUnique(const Expr& expr) {
-    return !(dg_.expr_node.at(expr)->parents.head &&
-             dg_.expr_node.at(expr)->parents.head->next);
+    auto it = dg_.expr_node.find(expr);
+    if (it == dg_.expr_node.end()) {
+      return true;
+    } else {
+      return !(it->second->parents.head && it->second->parents.head->next);
+    }
   }
 
+  // TODO(weberlo): Consolidate this method and `IsAtomic` in `pass_util.h`?
```

Comment:
Might be a good idea if we can avoid repeating ourselves, though are we certain that these definitions (something we will always inline and atomic expressions) will always coincide?

Reply:
Yeah.  I was originally thinking it might be a good idea, but I'm not convinced they should be merged anymore.